docs/.vuepress/.temp/pages/docs/atom-archive/getting-started/sections/why-atom.html.vue

```diff
@@ -0,0 +1,25 @@
+<template><div><h3 id="why-atom" tabindex="-1"><a class="header-anchor" href="#why-atom" aria-hidden="true">#</a> Why Atom?</h3>
```

Comment:
We should probably use a formatter for `*.html.vue` files, as this files illustrates

Reply:
Yea this isn't supposed to be here, .gitignore is screwing up